source/community/release_notes/sumac/dev_op_release_notes.rst

```diff
@@ -18,19 +18,53 @@ Breaking Changes
 ****************
 
```

Comment:
```suggestion

No breaking changes in Sumac.

```

I would add this line IF it is true, because otherwise reading the rendered page it's a bit unclear how the headers nest. But I want to check in on this - @arbrandes @bradenmacdonald @regisb @feanil - no breaking changes in Sumac? Who else should I tag?

Reply:
@feoh I didn't mean for you to commit this at this time. We need to check if there are any breaking changes.

Reply:
Crap itchy trigger finger. Unresolving. Will fix if people find breaking changes.

Reply:
> @feoh I didn't mean for you to commit this at this time. We need to check if there are any breaking changes.

I know, I'm sorry :) I'll fix it if people raise anything but I'm not going to create a revision to fix a thing which in fact may not be broken :) I'll stay on top of this and ensure the page reflects reality. Sorry for prematurely pulling the trigger!

Reply:
There is atleast https://github.com/openedx/edx-platform/pull/35185 which should be captured here as a breaking change.

Reply:
Oh, you're right, it was merged just after. Right in that case, I think there is nothing that I'm aware of that's breaking in Sumac.